tests/TTIR/test_compare.py

```diff
@@ -0,0 +1,65 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import pytest
+import jax
+import jax.numpy as jnp
+import numpy
+
+from infrastructure import verify_module
+
+# Note: TTNN does not support boolean data type, so bfloat16 is used instead.
+# Hence the output of comparison operation is bflaot16. JAX can not perform any
+# computation due to mismatch in output data type (in testing infrastructure).
+# The following tests explicitly convert data type of comparison operation
+# output for the verification purposes.
+
+
+def test_equal():
+    def module_equal(a, b):
+        c = a == b
+        return jax.lax.convert_element_type(c, jnp.float32)
+
+    verify_module(module_equal, [(64, 64), (64, 64)], dtype=jnp.bfloat16)
```

Comment:
Can you please add `parametrize` decorators to each test which take input shapes as parameter, like this:
```
@pytest.mark.parametrize("input_shapes", [[(64, 64), (64, 64)]])
def test_equal():
...
    verify_module(module_equal, input_shapes, dtype=jnp.bfloat16)
```